src/main/java/com/example/Library/controller/BooksController.java

```diff
@@ -33,15 +35,18 @@ public BooksController(BookService bookService) {
     public Page<Book> getAllBook(@RequestParam(value = "title") Optional<String> title,
                                  @RequestParam(value = "page") Optional<Integer> page,
                                  @RequestParam(value = "sortBy") Optional<String> sortBy) {
+        log.info("Executing method GET /api/v1/books with params : title = " + title + " page = " + page + " sortBy = " + sortBy);
         if (title.isPresent()) {
             return bookService.getBookByTitle(title.get());
         }
         return bookService.getAllBooksInPages(page.orElse(START_PAGE_BY_DEFAULT), sortBy.orElse(SORT_BY_DEFAULT_FIELD));
     }
+
     @PreAuthorize("hasAuthority('books:read')")
     @GetMapping("/{id}")
     @Operation(summary = "Get book by id")
     public Book getBook(@PathVariable long id) {
+        log.info("Executing method GET /api/v1/books/{id} with id " + id);
```

Comment:
pls try to avoid String concatenation during logging via Slf4j/Log4j, parametrized logging could be used instead https://www.tutorialspoint.com/slf4j/slf4j_parameterized_logging.htm